docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,348 @@
+$ cd alluxio-{{site.ALLUXIO_RELEASED_VERSION}}
+```
+
+This will create a directory `alluxio-{{site.ALLUXIO_RELEASED_VERSION}}` with all of the Alluxio
+source files and Java binaries.
+
+## Configure Alluxio
+
+Before we start Alluxio, we have to configure it. Please ensure all the subsequent commands are run on all the Linux nodes.
+
+In the `${ALLUXIO_HOME}/conf` directory, create the `conf/alluxio-site.properties` configuration
+file from the template.
+
+```bash
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
+
+Update `alluxio.master.hostname` in `conf/alluxio-site.properties` to the hostname of the machine
+you plan to run Alluxio Master on. Let's name it Alluxio Master node, and assume its hostname is
+**ALLUXIO_MASTER_HOSTNAME**.
+
+```bash
+$ echo "alluxio.master.hostname=ALLUXIO_MASTER_HOSTNAME" >> conf/alluxio-site.properties
+```
+
+### Configure for AWS
+
+Please ensure all the subsequent commands are run on all the Linux nodes.
+
+If you have an Amazon AWS account with your access key id and secret key, you can update your
+Alluxio configuration now in preparation for interacting with Amazon S3 later in this guide. Add
+your AWS access information to the Alluxio configuration by adding the keys to the
+`conf/alluxio-site.properties` file. The following commands will update the configuration.
+
+```bash
+$ echo "aws.accessKeyId=AWS_ACCESS_KEY_ID" >> conf/alluxio-site.properties
+$ echo "aws.secretKey=AWS_SECRET_ACCESS_KEY" >> conf/alluxio-site.properties
+```
+
+You will have to replace **AWS_ACCESS_KEY_ID** with your AWS access key id, and
+**AWS_SECRET_ACCESS_KEY** with your AWS secret access key. Now, Alluxio is fully configured for the
+rest of this guide.
+
+## Validate Alluxio environment
+
+Before starting Alluxio, you might want to make sure that your system environment is ready for running
+Alluxio services. You can run the following command to validate environment on all nodes:
+
+```bash
+$ ./bin/alluxio validateEnv all
+```
+
+You can also make the command run only specific validation task. For example,
+
+```bash
+$ ./bin/alluxio validateEnv all ulimit
+```
+
+Will only run validation tasks that check your system resource limits, on all the nodes.
+
+You can check out [this page](Developer-Tips.html) for detailed usage information regarding this command.
+
+## Set up a distributed storage as UFS
+
+The [local Getting-Started guide](Getting-Started.html) shows the steps of how to start Alluxio with default local file system.
+In a distributed cluster, Alluxio requires a distributed storage system as Under File System.
+In this doc, we use AWS S3 as an example here. If you want to use an existing distributed HDFS as UFS,
+please refer to this [doc](Configuring-Alluxio-with-HDFS.html).
+
+Also, in preparation for using S3 with Alluxio, create a bucket (or use an existing bucket). You
+should also note the directory you want to use in that bucket, either by creating a new directory in
```

Comment:
"Choose a directory in the S3 bucket to use as the under storage, creating it if it doesn't already exist"